aemedge/blocks/superhero/superhero.js

```diff
@@ -165,12 +172,14 @@ export default async function decorate(block) {
     ingestAnalytics(analyticsEvent);
   });
   // Get the rest of the images
-  assetIds.forEach(async (assetId, i) => {
-    if (i !== 0) {
-      const picture = await createPitcureFromAssetId(assetId, false, false, 'high');
-      if (picture !== null) imageContainer.append(picture);
-    }
-  });
+  setTimeout(() => {
```

Comment:
not sure this makes a difference, but just ensuring the block decode returns before these start loading.